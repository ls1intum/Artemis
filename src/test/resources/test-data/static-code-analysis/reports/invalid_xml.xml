<?xml version="1.0" encoding="utf-8"?>
<checkstyle version="4.3">
    <file name="/opt/bambooagent/bamboo-agent-home/xml-data/build-dir/MDTESTSWIFT-ARTEMISTESTUSER1-JOB1/assignment/Sources/swiftLib/Client.swift">
</checkstyle>
