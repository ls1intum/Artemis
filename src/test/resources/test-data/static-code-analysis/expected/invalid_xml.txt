{"tool":null,"issues":[{"filePath":"invalid_xml.xml","startLine":1,"rule":"ExceptionDuringParsing","category":"miscellaneous","message":"An exception occurred during parsing the report for file invalid_xml.xml. Exception: %s"}]}
